substrate/bin/node/runtime/src/lib.rs

```diff
@@ -3406,6 +3406,68 @@ impl_runtime_apis! {
 				key
 			)
 		}
+
+		fn trace_block(
+			block: Block,
+			config: pallet_revive::evm::TracerConfig
+		) -> Vec<(u32, pallet_revive::evm::CallTrace)> {
+			use pallet_revive::tracing::trace;
+			log::debug!(target: "runtime::revive", "Tracing block {:?}", block.header().number);
```

Comment:
Why only this api has logging? Is it left over. Generally I would say if you leave it in it should be `trace`. Same for the westend runtime.

Reply:
debugging left over will remove